Xcode/Examples/iPhoneXMPP/Classes/SettingsViewController.m

```diff
@@ -20,8 +20,8 @@ @implementation SettingsViewController
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 - (void)awakeFromNib {
-    [super awakeFromNib];
-    self.modalTransitionStyle = UIModalTransitionStyleFlipHorizontal;
+  [super awakeFromNib];
+  self.modalTransitionStyle = UIModalTransitionStyleFlipHorizontal;
```

Comment:
These two lines seem to be using two `space` instead of one `tab` for indentation